### PR TITLE
fix: support default message for teams only configs

### PR DESCRIPTION
**Pull request by rdmulford**
## Description
when there are no ‘messages’ which are basically prs from github and jira - it returns a default value for preparedBlocks which are posted as a part of the ‘main’ slack message, while the subChannelBlocks which are what gets posted for each team to each channel configured for that team is not set. In our case, we’re not defining any ‘main’ slack message and are just using the ‘teams’ config, so that default message is not posted due to the channel not being set, and the code never sets any default slack message for teams since the code exits before the team handling is done.

## How Has This Been Tested?

- [x] unit tests?

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore (updating repository quality of life, like deps, linting, etc.)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by jmccann** [comment]
Is there a test we can add for this?

File: src/utils/__tests__/prepareBlocks.test.js [hunk 1]
```diff
@@ -64,4 +64,59 @@ describe('createBlocks', () => {
...
+      name: 'FRIENDS',
+      blocks: Array(5).fill([createStorySection('hi')]),
+      issueTracking: {
+        jira: {
+          projects: ['CHANDLER', 'MONICA'],
+        },
+      },
+      channels: ['test-channel-1', 'test-channel-2'],
+    });
+    const blocks = createBlocks(false, {
```
Comment (jmccann) [inline src/utils/__tests__/prepareBlocks.test.js:82]: got confused on `false` here ... maybe we can set a variable with a more helpful name to pass in instead.  Sounds like this is where/how we provide "no messages" to `createBlocks`

Comment (rdmulford) [reply src/utils/__tests__/prepareBlocks.test.js]: fixed with https://github.com/target/captains-log/pull/157/commits/ca01e4905ba426833a8fc51823ab7d047e3e5f2f - using the same variable name as the function uses `hasMessages`

File: src/utils/__tests__/prepareBlocks.test.js [hunk 2]
```diff
@@ -64,4 +64,61 @@ describe('createBlocks', () => {
       }).preparedBlocks,
     ).toHaveLength(3);
   });
+
+  it('should create empty release for teams with channels when there are no messages', () => {
+    const hasMessages = false;
```
Comment (jmccann) [inline src/utils/__tests__/prepareBlocks.test.js:69]: Sorry! 🙇 I think this may be more clear:

```suggestion
    const hasNoMessages = false;
```

**Review by jmccann — APPROVED** [review]

**Pull request merged**